src/main/java/org/influxdb/dto/Point.java

```diff
@@ -116,7 +117,10 @@ private static void throwExceptionIfMissingAnnotation(final Class<?> clazz,
     public Builder tag(final String tagName, final String value) {
       Objects.requireNonNull(tagName, "tagName");
       Objects.requireNonNull(value, "value");
-      if (!tagName.isEmpty() && !value.isEmpty()) {
+      if (!tagName.isEmpty()
```

Comment:
Missing test coverage for the case where tag or value is nil or empty

Reply:
This will be added.